src/scenes/BlogPage/components/Previews/components/Switcher/Switcher.module.scss

```
@@ -72,48 +72,50 @@
 }
 
 .tabContentList {
-  display: grid;
-  grid-template-columns: 1fr 1fr 1fr;
-  grid-template-rows: repeat(auto-fill);
-  grid-gap: calc(40px * var(--coef-indent));
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  flex-wrap: wrap;
   list-style-type: none;
   padding: 0;
   margin: 0;
-
-  @media screen and (min-width: $lg) {
-    flex-wrap: wrap;
-    flex-direction: row;
-  }
-  @media screen and (max-width: $lg) {
-    grid-template-columns: 1fr 1fr;
-  }
-  @media screen and (max-width: $sm) {
-    grid-template-columns: 1fr;
-  }
 }
 
 .tabContentItem {
-  @include hl-height(500px);
-  position: relative;
+  flex: 1 1 calc(33.333% - 26.667px);
```

let's not use float numbers. Would it work like this
```suggestion
  flex: 1 1 calc(33.333% - 26px);
```
?